CHANGES.rst

```diff
@@ -1,12 +1,20 @@
 ``persistent`` Changelog
 ========================
 
-4.2.5 (unreleased)
+4.3.0 (unreleased)
```

Comment:
Is this a feature bump (seems like a bugfix to me).

Reply:
I went with a minor number bump because it's a C-programmer-visible change in semantics. It's subtle, but it's there, and I think there is a small chance it could break some previously working code.

But if it's a subtle enough change you'd prefer calling it a bug fix, I'm cool with that too :)

Reply:
I think I voted on #69 that anybody who can detect the change externally already Deserves to Lose(TM), but we can leave the bump in place if you want to be nice. :)

Reply:
I'm feeling generous today :)